impexp-core/src/main/java/org/citydb/core/operation/exporter/database/content/CityGMLExportManager.java

```diff
@@ -230,6 +235,10 @@ private AbstractGML processObject(AbstractGML object) throws CityGMLExportExcept
 				// trigger export of textures if required
 				if (isLazyTextureExport() && config.getExportConfig().getAppearances().isSetExportAppearance())
 					getExporter(DBLocalAppearance.class).triggerLazyTextureExport(feature);
+
+				if (idReplacer != null) {
+					feature.accept(idReplacer);
```

Comment:
This should not take place before putting the feature ids into the cache.  It'll cause empty cityobjectmembers when exporting CityObjectGroup

Reply:
Nice catch. The WFS, however, requires the replaced IDs in the cache (e.g., see [here](https://github.com/3dcitydb/web-feature-service/blob/fb4b38f238d2c2f3f8cb3dd4dce1e20f778589bc/src/main/java/vcs/citydb/wfs/operation/getfeature/citygml/CityGMLWriter.java#L314)). So, to fix this, both the original and the replaced IDs are now cached. See 5fca3c003621532650f3195f7e2baa2ebbaa8df6